### PR TITLE
Azure sentinel enhancement - `azure-sentinel-get-entity-by-id`

### PR DESCRIPTION
**Pull request by ChanochShayner**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
In case you are a technological partner:
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26936

## Description
Update the command "azure-sentinel-get-entity-by-id" to retrieve more information about the entity.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:408]: why is this hard coded?

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:414]: Shouldn't you combine the new field with the previous values? 
Are you currently just throwing the full object into the context?

File: Packs/AzureSentinel/ReleaseNotes/1_0_3.md
Comment (guykeller) [inline Packs/AzureSentinel/ReleaseNotes/1_0_3.md:4]:
```suggestion
- Updated the command ***azure-sentinel-get-entity-by-id*** to retrieve more information about the entity.
```

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [reply Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:414]: Moreover, why didn't we modify the yml as well?

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (ChanochShayner) [reply Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:408]: Take a quick look at this - https://techcommunity.microsoft.com/t5/azure-sentinel/get-entities-for-a-sentinel-incidient-by-api/m-p/1422643

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (ChanochShayner) [reply Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:414]: We do not have events in the system to check what is coming back.
That's why I'm not entirely sure of my answers.
I think the current implementation also has the information that was in the previous implementation and more information.

**Review by guykeller — APPROVED** [review]

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [reply Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:414]: Why did you resolve this? @ChanochShayner 
@dantavori Would love your advice here

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (dantavori) [reply Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:414]: @ChanochShayner this might help:
* current command's outputs - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/entities/GetSecurityAlertEntityById.json
* `/expand` outputs - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/entities/expand/PostExpandEntity.json

note that the `/expand` example it's an *account* entity (and not a security alert), but if the structure is the same for every entity type then it looks like the two responses above have the same structure (id, name, type, kind and properties) and what differentiates them is the "properties" value (I guess each one has different attributes). So maybe the right thing to do is to combine the properties from the two responses.

However, this is just an advice and the best thing to do is to try the endpoint with data. Did you manage to get something with @Arsenikr ?

**Review by guykeller — CHANGES_REQUESTED** [review]

**Comment by ChanochShayner** [comment]
@dantavori @guykeller There was a meeting with the customer this morning,
The part we added to the integration works well.
Ready for review.
Thanks.

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:101]:
```suggestion
    return string_var.lower() == 'true'
```

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:100]: `string_var` - not a good name

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py
Comment (guykeller) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:436]: Please add a comment explaining why this should be hard coded like this

**Review by guykeller — APPROVED** [review]
Approving with notes - please fix them before merging

**Pull request merged**
